### PR TITLE
Some small updates

### PR DESCRIPTION
**Pull request by olivierpaul**

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (ahadas) [inline gradlew]: I'm more than ok with upgrading gradle, but do we need these options?

File: gradlew.bat [hunk 2]
```diff
@@ -14,7 +30,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m" "-Xms64m"
```
Comment (ahadas) [inline gradlew.bat]: same here

**Comment by olivierpaul** [comment]
Hello, this is the new default value when you update your project to the
latest version of Gradle

**Comment by ahadas** [comment]
> Hello, this is the new default value when you update your project to the latest version of Gradle Le lun. 6 mai 2019 à 18:25, Arik Hadas <notifications@github.com> a écrit :
> […](#)

Oh I see.
How did you upgrade? With `./gradlew wrapper --gradle-version=5.4.1 --distribution-type=bin` I see that only `-Xmx64m` is added

**Comment by olivierpaul** [comment]
Exactly :-)

**Comment by ahadas** [comment]
@olivierpaul Looks great now, thank you for this contribution!

**Pull request merged**